app/images/warning.svg

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
need to optimized this SVG, can just use https://jakearchibald.github.io/svgomg/